src/Aside/DateCells.tsx

```diff
@@ -0,0 +1,66 @@
+import { isToday } from '../utils';
+import styles from './aside.module.css';
+const TOTAL_CELLS = 42;
+
+interface DateCellsProps {
+  date: Date;
+}
+
+const DateCells = ({ date }: DateCellsProps) => {
+  const displayedMonth = new Date(date.getFullYear(), date.getMonth(), 1);
+
+  const firstDayOfMonth = new Date(
+    displayedMonth.getFullYear(),
+    displayedMonth.getMonth(),
+    1
+  );
+  let startDay = firstDayOfMonth.getDay();
+  if (startDay === 0) startDay = 7;
+  const lastDayOfMonth = new Date(
+    displayedMonth.getFullYear(),
+    displayedMonth.getMonth() + 1,
+    0
+  );
+  const totalDays = lastDayOfMonth.getDate();
+  const previousMonthLastDay = new Date(
+    displayedMonth.getFullYear(),
+    displayedMonth.getMonth(),
+    0
+  );
+  const previousMonthTotalDays = previousMonthLastDay.getDate();
```

Comment:
getLastMonthTotalDays(date) <- where date is current date, eg July, but function will return June total days